src/status_im/ethereum/json_rpc.cljs

```diff
@@ -55,6 +55,7 @@
    ;;TODO not used anywhere?
    "shhext_deleteChat" {}
    "shhext_saveContact" {}
+   "shhext_verifyENSNames" {}
```

Comment:
I thought this would entirely be done on go side?

Reply:
No, eventually maybe, but we are not parsing messages yet in status-go (we only do for the console client, but otherwise we would be parsing them twice, as we still have to port some of the handling, so it's disabled in status-react), and the ens name is inside the `content` field (or contact-request).
So status-react will parse the message and send the usernames to be verified to status-go (note that even if we did resolve them in status-go, it would be asynchronous, as resolving username can take up to 1,2 seconds, so we would pass the messages immediately and send another signal once resolved, so from a user perspective it would be asynchronous in both cases)